README.md

```diff
@@ -55,18 +55,20 @@ Create a new file in `src/test/scala/`, for example, `BasicTest.scala`.
 In this file:
 1.  Add the necessary imports:
     ```scala
-    import org.scalatest._
-    import chiseltest._
-    import chisel3._
+   import org.scalatest._
+   import org.scalatest.flatspec._ 
+   import matchers.should._
+   import chiseltest._
+   import chisel3._
     ```
 2.  Create a test class:
     ```scala
-    class BasicTest extends FlatSpec with ChiselScalatestTester with Matchers {
+    class BasicTest extends AnyFlatSpec with ChiselScalatestTester with Matchers {
```

Comment:
If previous review is accepted, this should be changed to:
`class BasicTest extends AnyFlatSpec with ChiselScalatestTester with should.Matchers {`

Reply:
I'm going to use Kevin's suggested imports. I'll add something to the `README` in this pr about `Matchers` being an optional addition.